frontend/src/components/modules/Tournaments/Signup/Signup.tsx

```diff
@@ -115,6 +134,23 @@ const Signup: React.FC = (): ReactElement => {
         </Box>
       </Box>
 
+      {/* Password Input if Tournament is Private */}
+      {Boolean(tournament.passwordEnabled) && (
```

Comment:
No need to fix just wondering why this is different from some other places. (not enabled === true)